modules/config/pages/eap-cli.adoc

```diff
@@ -1,5 +1,189 @@
 = JBoss EAP CLI
```

Comment:
I am assuming this file is just your high level plan and the content is not part of the PR. SO I am going to  ignore this...